README.md

```diff
@@ -104,19 +104,20 @@ Note on components value tolerance:
 12 components on the Right side of SATA port, highlighted in green:<br>
 ![](https://github.com/badger707/m920q-dual-NVME/blob/main/pictures/IMG_5878_x.png)
 <br><br>
-<b>For M920Q</b> -- move resistor from R105 to R162. This will switch PCH to work in M920X mode:
+<b>For M920Q</b> -- move resistor from R150 to R162. This will switch PCH to work in M920X mode:
 ![](https://github.com/badger707/m920q-dual-NVME/blob/main/pictures/IMG_5883_x.png)
 <br><br>
-<b>For M720Q</b> -- move resistor from R151 to R157. This will switch PCH to work in M920X mode:
+<b>For M720Q</b> -- move resistor from R151 to R157. <i>Note that an <b>NEC</b> B360 chipset may require R152 moved to R159 instead.</i> This will switch PCH to work in M920X mode:
 ![](https://github.com/badger707/m920q-dual-NVME/blob/main/pictures/m720q_pch_jump.jpeg)
 <br>
 Note on above M720Q picture components location:
-> I can not confirm this will work 100%, I have no M720Q on hands to do real test, however, looking at pictures provided by Redditor and comparing it with M920X schematics -- this should work just fine. Create an Issue if this need further checks.
+> I can not confirm this will work 100%, I have no M720Q on hands to do real test, however, looking at pictures provided by Redditor and comparing it with M920X schematics -- this should work just fine. Create an issue if this need further checks.
 <br><br>
 * ### Component location - BOTTOM side of the board
 
-Only 4 capacitors at the side of NVME connector pads, highlighted in green:<br>
+Only 5 capacitors at the side of NVME connector pads, highlighted in green:<br>
```

Comment:
thats not correct, 4 caps, not 5

Reply:
This was only updated due to RC26. Will resolve.